lua/autorun/client/cl_hideholos.lua

```diff
@@ -4,7 +4,7 @@ net.Receive( "update_hologram_visibility_preference", function( len )
 end )
 
 net.Receive( "get_user_hologram_visibility_preference", function( len )
-    local chip = cookie.GetNumber( "cfc_hologram_visibility_preference", 0 ) ~= 0 and true or false
+    local chip = cookie.GetNumber( "cfc_hologram_visibility_preference", 0 )
```

Comment:
I think you'll still need the ~=0, lua takes `0` as `true`
```suggestion
    local chip = cookie.GetNumber( "cfc_hologram_visibility_preference", 0 ) ~= 0
```

Reply:
Really? That’s weird

Reply:
Yeah, only `false` and `nil` evaluate to `false`